network/load-balancer/how-to/add-certificate.mdx

```diff
@@ -0,0 +1,162 @@
+
+7. Click the **Frontends** tab.
+
+8. Click the three dot menu next to the frontend you want to add the certificate to, then click **Edit**
+
+    <Lightbox src="scaleway-edit-frontend.webp" alt="" />
+
+9. Select the certificate you just created from the **SSL Certificate(s)** dropdown, then click **Edit Frontend**.
+
+Your certificate is added to your Load Balancer's frontend.
+
+## How to create a self-signed certificate
+
+If you are using the [import certificate](#how-to-import-a-certificate) option, you may wish to generate your own self-signed certificate to import to your Load Balancer. These certificates are not signed by any publically-trusted Certificate Authority, but may be suitable for development ad testing purposes, or internal use. A self-signed certificate can be issued easily on any computer using the `openssl` tool. Follow these steps to generate a self-signed certificate:
+
+1. From your computer, open up a command line terminal and run the following command to generate a private key file and a CSR file:
+	```
+	openssl req -new -newkey rsa:4096 -nodes -keyout private_key.txt -out csr.txt -subj "/C=FR/ST=Ile-de-France/L=Paris/O=MyInternetCompanyLTD/CN=www.example.com"
+	```
+	The following subcommands are used with the `openssl` base command:
+	- `req`: This subcommand specifies the use of the X.509 certificate signing request (CSR) management.
+	- `newkey rsa: 4096`:  This subcommand specifies the creation of a new key and certificate at the same time using a 4096 bit long RSA key.
+	- `nodes`: This option tells OpenSSL to skip the securisation of the certificate using a passphrase.
+	- `keyout`: This subcommand defines the location and file name of the private key file
+	- `out`: This specifies the path and file name of the generated certificate request.
+	- `sub`: The subject for the certficate. Modify the values of these subcommand to your requirements. The **Common Name** (CN) represents the domain name you are issuing the certificate for.
+
+2. Generate a file containing all **Subject Alternative Names** (SAN) for the certificate. They include websites, IP addresses, common names, etc. They are protected by a single SSL Certificate. 
+
+3. Create a new file called `alt_names.txt`, open it in a text editor and edit the file as follows. Make sure to add all domain names and IP addresses that should be protected by the certificate.
+	```
+	subjectAltName=DNS:example.com,DNS:www.example.com
+	```
+
+4. Save the file and exit the text editor.
+
+5. Generate the self-signed certficate using the `openssl` tool:
+	```
+	openssl x509 -req -extfile alt_names.txt -sha256 -days 365 -in csr.txt -signkey private_key.txt  -out certificate.txt
+	```
+
+6. Check that you have the following files in your folder with the `ls` command.
+	- `csr.txt`: The certficate request
+	- `private_key.txt`: The private key for the certificate
+	- `certificate.txt`: The public key for the certificate
+
```

Comment:
I believe we are missing a step and should explain how to combine the certificate and key into a pem file since this is the format that we require in the console to add the certificate to the load balancer.

Reply:
I've added what I think/hope is the correct action, OK for you?

Reply:
Sounds good, just added a comment to reorder the command line